config-connector/solutions/projects/kpt/owned-project/README.md

```diff
@@ -0,0 +1,66 @@
+Owned Project
+==================================================
+
+# NAME
+
+  owned-project
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create
+  a project in a folder, binding an IAM member
+  as project owner.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  Run `kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/projects/kpt/project-owner project-owner`.
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a
+  service account with the following roles in
+  the organization:
+  - roles/resourcemanager.folderViewer
+  - roles/resourcemanager.projectCreator
+  - roles/iam.securityAdmin
+
+# USAGE
+
+  Replace the
+  `${BILLING_ACCOUNT_ID?}` value:
+
+  From within this directory, run
+  ```
+  kpt cfg set . billing-account VALUE
+  ```
+  replacing `VALUE` with your billing account
+  ID.
+
+  Replace the `${FOLDER_ID}` the same way, using:
+  ```
+  kpt cfg set . folder-id VALUE
+  ```
+  where VALUE is the numeric folder ID of the folder to create the new project under.
+
+  You will need to reset the project ID,
```

Comment:
"reset" is confusing. Sounds like it's already set. Maybe say "You'll also need to configure the project ID, ..."